Doc/library/os.rst

```diff
@@ -2121,7 +2121,11 @@ features:
 
    .. audit-event:: os.chmod path,mode,dir_fd os.lchmod
 
-   .. availability:: Unix.
+   .. availability:: Unix, not Linux, FreeBSD >= 1.3, NetBSD >= 1.3, not OpenBSD
```

Comment:
Nice but is not rendered: https://cpython-previews--107960.org.readthedocs.build/en/107960/library/os.html#os.lchmod

Reply:
Maybe because of the commit/force merge mess I did? It seems to have been generated successfully now?!

Reply:
The build should not care at all about git operations, the latest build for the latest commit gets published at the address.

More likely the text after the comma is just ignored.  Can you check the code in the original PR that added the availability directive to find out?

Reply:
@merwok it looks rendered to me?

![image](https://github.com/python/cpython/assets/9087854/9431a0e1-8703-41c3-b10c-cb1273d7ad16)

Reply:
Ah great!  Maybe I opened the link before that commit was processed.